### ISSUE TITLE
Cant find /etc/g14-perf-control.toml

### ISSUE THREAD

**Issue by nsk126** [m0ppers/g14-perf-control#3]
I have an Ubuntu 22.04 install. I don't feel my fans blowing at all but the output for `./g14-perf-control info` is `Fan profile: normal, Turbo: true`

I tried `sudo ./g14-perf-control set Turbo` and got Error: ````Failed to read configfile `/etc/g14-perf-control.toml` ````

looks like i don't even have a `/etc/g14-perf-control.toml` file.

I have a Ryzen 9 6900HS model of the G14 2022 series. Any idea whats the problem?

**Comment by m0ppers** [comment]
hi. you need to create the config file. an example file can be found here `https://github.com/m0ppers/g14-perf-control/blob/main/example_config.toml`

`g14-perf-control info` only reads the current state from the kernel settings so that works without the config file. but setting stuff depends on the file.

the tool can't read the current fan curves  unfortunately. if nothing was set then you probably are using defaults that were set during poweron (whatever ASUS assumed to be sane defaults)

**Comment by m0ppers** [comment]
additional note: I have the 2020 model. I can't guarantee that ASUS changed something. it SEEMS that it still works the same way as the info command returned something sensible.

**Comment by nsk126** [comment]
I used the same contents in the `example_config.toml` for the `/etc/g14-perf-control.toml` file. Still doesn't seem to work. The binary I built returns 

```
Error: Couldn't change profile

Caused by:
    Invalid Board. Are you using a G14?
```
I downloaded the binary from your v0.1.1 realease, that returns the same thing too.

**Comment by m0ppers** [comment]
yeah that is probably related to the 2022 version. what does the following command show

```
cat /sys/class/dmi/id/board_name
```

maybe I can push an update and make it 2022 compatible :partying_face:

**Comment by nsk126** [comment]
`GA402RK`

**Comment by nsk126** [comment]
as a side note, I find it really odd that my fans are always at 0RPM i.e I don't hear any sound.
I checked this with `sensors | grep -i fan` which returns
```
fan1:           0 RPM  (min =    0 RPM, max = 3300 RPM)
cpu_fan:        0 RPM
```
My temps are always peaking just with chrome running lol.

**Comment by m0ppers** [comment]
hmm the underlying library I am using seems also unmaintained :D I have created a patch. can you try this one here: 

https://github.com/m0ppers/g14-perf-control/releases/tag/v0.1.2

As I said. I have no idea if this really works on a 2022 model. this is just a wild guess.

Make sure you have the acpi call module installed. this should work: https://packages.ubuntu.com/jammy/acpi-call-dkms

**Comment by nsk126** [comment]
Downloaded the apt package `acpi-call-dkms` and your new release file.

Still getting this message though on calling `sudo ./g14-perf-control set boost`
```
Error: Couldn't change profile

Caused by:
    0: failed to write to file No such file or directory (os error 2)
    1: No such file or directory (os error 2)
```

**Comment by m0ppers** [comment]
does `/proc/acpi/call` exist? if not then you need to additionally do a `modprobe acpi_call`. if it does exist then it is probably some serious change that asus made. I am afraid I can only redirect you to the asus linux tools for setting fan control etc : https://asus-linux.org/. I don't fell comfortable to make changes blindly :cry:

**Comment by nsk126** [comment]
I get `modprobe: FATAL: Module acpi_call not found in directory /lib/modules/5.15.0-57-generic`.

Also - my apologies for late reply.
